src/main/java/at/hannibal2/skyhanni/utils/RegistryUtil.kt

```diff
@@ -0,0 +1,56 @@
+package at.hannibal2.skyhanni.utils
+
+object RegistryUtil {
```

Comment:
Probably not super high prio, but shelling out is usually pretty expensive. Maybe we can apply some sort of caching layer to the functions in here? Depends on how often these functions are called.